cycode/cyclient/scan_client.py

```diff
@@ -72,17 +72,27 @@ def zipped_file_scan(
 
         return self.parse_zipped_file_scan_response(response)
 
-    def get_zipped_file_scan_async_url_path(self, scan_type: str) -> str:
+    def get_scan_report_url(self, scan_id: str, scan_type: str) -> models.ScanReportUrlResponse:
+        response = self.scan_cycode_client.get(url_path=self.get_scan_report_url_path(scan_id, scan_type))
+        return models.ScanReportUrlResponseSchema().build_dto(response.json())
+
+    def get_zipped_file_scan_async_url_path(self, scan_type: str, should_use_scan_service: bool = False) -> str:
```

Comment:
- `get_zipped_file_scan_async_url_path` method is used only in `zipped_file_scan_async`
- `zipped_file_scan_async`  method is used only in `perform_scan_async`

`perform_scan_async` performs scans via scan service only. no need for `should_use_scan_service` here